server/server.go

```diff
@@ -399,10 +399,12 @@ func (s *server) updateClusterInfo() {
 	for chainID, chainInfo := range s.network.customChainIDToInfo {
 		s.clusterInfo.CustomChains[chainID.String()] = chainInfo.info
 	}
-	s.clusterInfo.Subnets = s.network.subnets
-	s.clusterInfo.SubnetParticipants = make(map[string]*rpcpb.SubnetParticipants)
-	for subnetID, nodes := range s.network.subnetParticipants {
-		s.clusterInfo.SubnetParticipants[subnetID] = &rpcpb.SubnetParticipants{NodeNames: nodes}
+	s.clusterInfo.Subnets = make(map[string]*rpcpb.SubnetInfo)
```

Comment:
pls do this for s.network.subnets inside of `server/network.go` and then just copy `s.clusterInfo.Subnets = s.network.subnets`

Reply:
or make a copy similar to CustomChains, but let create the rpcpb objects in `server/network.go` as the other two fields